OOps/midiops2.c

```diff
@@ -323,9 +323,9 @@
     if (UNLIKELY((ctlno = (int32) *p->ictlno) < 0 || ctlno > 127)) {
       return csound->InitError(csound, Str("illegal controller number"));
     }
-    else if (UNLIKELY((chan=(int32_t) *p->ichan-1) < 0 || chan > 15)) {
+    else if (UNLIKELY((chan=(int32_t) *p->ichan-1) < 0)) {
```

Comment:
If I understand correctly, the max channel is MIDIMAXPORTS * 16 = 1024, so we may want to add back the upper bounds check for < 1024 throughout this PR. What do you think @vlazzarini?

Reply:
yes, good idea